niche_vlaanderen/nutrient_level.py

```diff
@@ -45,7 +60,7 @@ def __init__(
         self._ct_nutrient_level = pd.read_csv(ct_nutrient_level)
         self._ct_soil_code = pd.read_csv(ct_soil_code)
 
-        # convert the mineralisation to float so we can use np.nan for nodata
+        # convert the mineralisation system table to float to use np.nan for nodata
         self._ct_mineralisation["nitrogen_mineralisation"] = self._ct_mineralisation[
             "nitrogen_mineralisation"
         ].astype("float64")
```

Comment:
you didn't change this, but I wonder if we should not convert to float32 if we use that for the mineralisation raster.

Reply:
good idea!